### PR TITLE
update to 5.5.0.RC2-SNAPSHOT #729

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #729.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <parent>
     <groupId>org.terasoluna.gfw</groupId>
     <artifactId>terasoluna-gfw-parent</artifactId>
-    <version>5.5.0-SNAPSHOT</version>
+    <version>5.5.0.RC-SNAPSHOT</version>
```
Comment (yoshikawaa) [inline pom.xml]: - [ ] Wrong version... Please fix to `5.5.0.RC2-SNAPSHOT`.

Comment (btkobayashirun) [reply pom.xml]: 修正しました。

c5b371ecf0d0faad6d61544213b1c7a004bf8b54#diff-600376dffeb79835ede4a0b285078036R18

File: pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
   <parent>
     <groupId>org.terasoluna.gfw</groupId>
     <artifactId>terasoluna-gfw-parent</artifactId>
-    <version>5.5.0-SNAPSHOT</version>
+    <version>5.5.0.RC2-SNAPSHOT</version>
```
Comment (yoshikawaa) [inline pom.xml]: - [ ] Please also change branch name at [travis.yml](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/c5b371ecf0d0faad6d61544213b1c7a004bf8b54/.travis.yml#L15).

Comment (yoshikawaa) [reply pom.xml]: I checked to fix at 9f85b26 , thanks @btkobayashirun .

**Comment by yoshikawaa** [comment]
@btkobayashirun 
There is a build error, but I will merge.
Let's create and correspond to Issue every error, and finally eliminate all errors and create PR for master branch.

**Pull request merged**